tests/drivers/uart/uart_basic_api/src/test_uart_config.c

```diff
@@ -36,7 +36,7 @@ const struct uart_config uart_cfg = {
 
 static int test_configure(void)
 {
-	const struct device *uart_dev = device_get_binding(UART_DEVICE_NAME);
+	const struct device *uart_dev = DEVICE_DT_GET(DT_CHOSEN(zephyr_console));
 
 	if (!uart_dev) {
```

Comment:
The `!uart_dev` change to `!device_is_ready(uart_dev)` needs to be done for all the changes in this commit as well.